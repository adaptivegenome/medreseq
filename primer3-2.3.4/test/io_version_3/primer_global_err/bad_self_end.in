PRIMER_SEQUENCE_ID=bad_self_end
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_SELF_END=-1
=
