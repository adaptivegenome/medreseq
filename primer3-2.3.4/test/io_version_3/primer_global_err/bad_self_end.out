PRIMER_SEQUENCE_ID=bad_self_end
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_SELF_END=-1
PRIMER_ERROR=Illegal value for primer complementarity restrictions
=
